sdk/webpubsub/Azure.Messaging.WebPubSub/src/GlobalSuppressions.cs

```diff
@@ -4,3 +4,4 @@
 using System.Diagnostics.CodeAnalysis;
 
 [assembly: SuppressMessage("Usage", "AZC0002:DO ensure all service methods, both asynchronous and synchronous, take an optional CancellationToken parameter called cancellationToken.", Justification = "CancellationToken can be passed through RequestOptions")]
+[assembly: SuppressMessage("Usage", "AZC0018: Protocol method should have requestContext as the last parameter and don't have model as parameter type or return type. Protocol method should not have optional parameters if ambiguity exists between protocol method and convenience method.", Justification = "CancellationToken can be passed through RequestOptions")]
```

Comment:
I don't think we should be adding global suppressions to libraries.

Reply:
Let's ask a simple question: is AZC0018 implemented correctly? 

If so, then webpubsub not follows AZC0018 and it should be suppressed.

Reply:
That is a fair question, @pshao25 -- would you share the answer?

I think there are a couple more questions worth asking here as well:
- What is the goal of having AZC0018?
- Why does WebPubSub have methods that are violating it?